app/graphql/types/survey_type.rb

```diff
@@ -9,9 +9,10 @@ class SurveyType < Types::Base::Object
     field :thank_email_below_threshold, String, null: true
     field :is_active, Boolean, null: true
     field :cover_image_url, String, null: true
-    field :created_at, String, null: true
-    field :active_at, String, null: true
-    field :inactive_at, String, null: true
-    field :type, String, null: true
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: true
+    field :active_at, GraphQL::Types::ISO8601DateTime, null: true
+    field :inactive_at, GraphQL::Types::ISO8601DateTime, null: true
+    field :survey_type, String, null: true, method: :type
```

Comment:
Is this the same field `type` as before? What is `method: :type` for? IS it for the pagination? I am confused here 😵